snapshot/snapshotgc/gc.go

```diff
@@ -184,50 +184,52 @@ func findUnreferencedAndRepairRereferenced(
 		return nil
 	})
 
-	unusedCount, unusedBytes := unused.Approximate()
-	inUseCount, inUseBytes := inUse.Approximate()
-	systemCount, systemBytes := system.Approximate()
-	tooRecentCount, tooRecentBytes := tooRecent.Approximate()
-	undeletedCount, undeletedBytes := undeleted.Approximate()
-
-	userLog(ctx).Infof("GC found %v unused contents (%v)", unusedCount, units.BytesString(unusedBytes))
-	userLog(ctx).Infof("GC found %v unused contents that are too recent to delete (%v)", tooRecentCount, units.BytesString(tooRecentBytes))
-	userLog(ctx).Infof("GC found %v in-use contents (%v)", inUseCount, units.BytesString(inUseBytes))
-	userLog(ctx).Infof("GC found %v in-use system-contents (%v)", systemCount, units.BytesString(systemBytes))
-	userLog(ctx).Infof("GC undeleted %v contents (%v)", undeletedCount, units.BytesString(undeletedBytes))
-
-	contentlog.Log2(ctx, log,
-		"GC found unused contents",
-		logparam.UInt32("count", unusedCount),
-		logparam.Int64("bytes", unusedBytes))
-	contentlog.Log2(ctx, log,
-		"GC found unused contents that are too recent to delete",
-		logparam.UInt32("count", tooRecentCount),
-		logparam.Int64("bytes", tooRecentBytes))
-	contentlog.Log2(ctx, log,
-		"GC found in-use contents",
-		logparam.UInt32("count", inUseCount),
-		logparam.Int64("bytes", inUseBytes))
-	contentlog.Log2(ctx, log,
-		"GC found in-use system-contents",
-		logparam.UInt32("count", systemCount),
-		logparam.Int64("bytes", systemBytes))
-	contentlog.Log2(ctx, log,
-		"GC undeleted contents",
-		logparam.UInt32("count", undeletedCount),
-		logparam.Int64("bytes", undeletedBytes))
+	result := buildGCResult(&unused, &inUse, &system, &tooRecent, &undeleted, &deleted)
+
+	userLog(ctx).Infof("Snapshot GC statistics: %v", result)
+	contentlog.Log1(ctx, log, "Snapshot GC", result)
```

Comment:
This is sending the `results` only to the `contentlog`.
Before, there was also an informational message, with human-readable units in the `userLog`. It may make sense to keep the message in `userLog`.

Reply:
Added the human-readable userlog back.